README.md

```diff
@@ -330,15 +330,22 @@ _Source:_ [What is Docker](https://www.docker.com/why-docker)
 
 ### User Interface
 
+#### IDE integrations
```

Comment:
I think it's a good idea to have a section for IDEs here, but it could be highly opinionated (but so is this list).
What do you think @veggiemonk / @agebhar1?

Reply:
Well, you may wonder how many peoples don't even thought that their IDE can work with Docker.
I created a separate commit with IDE integrations so you may cherry pick only the first commit.

Reply:
Good idea! 👍